crates/sui-types/src/error.rs

```diff
@@ -563,6 +585,7 @@ pub enum VMMemoryLimitExceededSubStatusCode {
 }
 
 pub type SuiResult<T = ()> = Result<T, SuiError>;
+pub type UserInputResult<T = ()> = Result<T, UserInputError>;
 
 // TODO these are both horribly wrong, categorization needs to be considered
```

Comment:
@tnowacki how worried should we be about this comment?

Reply:
what are horribly wrong?

Reply:
Oh my goodness. Yeah these are _REALLY_ bad. We should never jump directly from a VMError to a SuiError. I imagine it probably isn't used anywhere anymore? (at least I hope so)